### PR TITLE
Feat/quiz category

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added Quiz Categories 

## Screenshot 
image

## Files to be ignored while reviewing

- assets/images

## Preview Link:
https://deploy-preview-2--guessthisquiz.netlify.app/

### CODE REVIEW

File: src/pages/QuizCategory/QuizCategory.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      <div className="quiz-container flex-center">
+        <div className="grid-container flex-spbt">
+          <div className="row">
+            <h1>Let's Play</h1>
+          </div>
+        </div>
+      </div>
+      <div className="category-container flex-spbt">
+        {categories.map((category, index) => (
+          <Card title={category.title} image={category.image} key={index + 1} />
```
Comment (Naveenchandar) [inline src/pages/QuizCategory/QuizCategory.jsx:15]: Any specific reason for using index as key ? . Id key is already present in categories data ?. Please use id as key.

File: src/utils/set-title.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const setTitle = (title) => {
+  document.title = title;
+};
+
+export { setTitle };
```
Comment (Naveenchandar) [inline src/utils/set-title.js:5]: If needed, you can convert it to custom hooks and pleas ignore if not needed.

**Review by Naveenchandar — APPROVED** [review]
Left few comments, 
Neat folder structure
Good usage of theme context
Maintained file name consistency

File: src/pages/QuizCategory/QuizCategory.jsx (hunk 1 above)
Comment (Aditi0505) [reply src/pages/QuizCategory/QuizCategory.jsx:15]: Thanks for pointing it out. I will change this.

File: src/utils/set-title.js (hunk 2 above)
Comment (Aditi0505) [reply src/utils/set-title.js:5]: I will implement this later on. I have simply used a function as I have not set the title in a state.

**Review by anupkgurung — APPROVED** [review]
good use of components.
Clean code.
svg or any file that need no review can be avoided while commit or can be directly pushed to dev.

**Pull request merged**